lib/flutter_calendar_carousel.dart

```diff
@@ -1136,24 +1142,32 @@ class _CalendarState<T extends EventInterface>
       bool isNextMonthDay,
       bool isThisMonthDay,
       DateTime now) {
-    final customDayBuilder = widget.customDayBuilder;
+    final customDayBuilder = this.widget.customDayBuilder;
```

Comment:
Why changing this to `this`?

Reply:
Because a variable named `widget` exists in the same scope (even if declared later). It won't compile unless explicitly referenced through `this`.

Reply:
Writing `this` inside the stateful widget looks a bit awkward here. How about changing the above variable name to something else?

Reply:
It's the same to me, it is all within a small method. Should I change `Widget widget` to `Widget tmp` and remove the `this` prefix? I see `tmp` is used in other places.

Reply:
How about `eventWidget`?

Reply:
I went with `dayContainer`, it matches the method name and what it is trying to create.

Anything else I can fix with this?

Reply:
@hyochan poke! :)